client/src/app/site/motions/components/motion-change-recommendation/motion-change-recommendation.component.ts

```diff
@@ -104,14 +104,16 @@ export class MotionChangeRecommendationComponent {
     public createForm(): void {
         this.contentForm = this.formBuilder.group({
             text: [this.changeReco.text, Validators.required],
-            diffType: [this.changeReco.type, Validators.required]
+            diffType: [this.changeReco.type, Validators.required],
+            public: [!this.changeReco.internal]
         });
     }
 
     public saveChangeRecommendation(): void {
         this.changeReco.updateChangeReco(
             this.contentForm.controls.diffType.value,
-            this.contentForm.controls.text.value
+            this.contentForm.controls.text.value,
+            !this.contentForm.controls.public.value
```

Comment:
why the `!`?

Reply:
Same as above. For the UI I find a checkbox that sets it as public more intuitive than the provided "out-out"-field "internal".